### PR TITLE
Manual updates 20220515 dependencies

### PR DESCRIPTION
**Pull request by moljac**
### Does this change any of the generated binding API's?

Possibly. Updates for dependencies.

### Describe your contribution

- bound updated/bumped dependencies

- com.google.firebase:firebase-encoders-proto -  -> 16.0.0
- com.github.bumptech.glide:annotations - 4.12.0 -> 4.13.2
- com.github.bumptech.glide:disklrucache - 4.12.0 -> 4.13.2
- com.github.bumptech.glide:gifdecoder - 4.12.0 -> 4.13.2
- com.github.bumptech.glide:glide - 4.12.0 -> 4.13.2
- com.github.bumptech.glide:recyclerview-integration - 4.12.0 -> 4.13.2
- com.google.android.datatransport:transport-backend-cct - 3.0.0 -> 3.1.4
- com.google.android.datatransport:transport-runtime - 3.0.1 -> 3.1.4
- com.google.dagger:dagger - 2.40.5 -> 2.41
- com.google.errorprone:error_prone_annotations - 2.5.1 -> 2.6.0
- com.google.zxing:core - 3.4.1 -> 3.5.0
- com.google.protobuf:protobuf-javalite - 3.15.8 -> 3.16.1
- io.grpc:grpc-android - 1.28.0 -> 1.29.0
- io.grpc:grpc-api - 1.28.0 -> 1.29.0
- io.grpc:grpc-context - 1.28.0 -> 1.29.0
- io.grpc:grpc-core - 1.28.0 -> 1.29.0
- io.grpc:grpc-okhttp - 1.28.0 -> 1.29.0
- io.grpc:grpc-protobuflite - 1.28.0 -> 1.29.0
- io.grpc:grpc-stub - 1.28.0 -> 1.29.0
- io.opencensus:opencensus-api - 0.28.3 -> 0.29.0
- io.opencensus:opencensus-contrib-grpc-metrics - 0.28.3 -> 0.29.0
- io.perfmark:perfmark-api - 0.23.0 -> 0.24.0
- org.chromium.net:cronet-api - 76.3809.111 -> 92.4515.131
- org.chromium.net:cronet-common - 76.3809.111 -> 92.4515.131
- org.chromium.net:cronet-embedded - 76.3809.111 -> 92.4515.131
- org.chromium.net:cronet-fallback - 76.3809.111 -> 92.4515.131
- org.codehaus.mojo:animalsniffer-annotations - 1.18 -> 1.19
- org.tensorflow:tensorflow-lite - 2.5.0 -> 2.6.0
- org.tensorflow:tensorflow-lite-gpu - 2.5.0 -> 2.6.0

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1412,8 +1420,8 @@
       {
         "groupId": "com.squareup.okio",
         "artifactId": "okio",
-        "version": "2.10.0",
-        "nugetVersion": "2.10.0.3",
+        "version": "3.0.0",
```
Comment (jpobst) [inline config.json:1423]: It looks like this version uses some new packaging format that our tooling does not support.  This `.jar` is empty, and thus the package we are making for it is empty.  The resulting NuGet will not contain C# or Java code and will not be useful for our users.

Let's hold off on updating this package until we can correctly support it.

The actual implementation may have moved to https://mvnrepository.com/artifact/com.squareup.okio/okio-jvm/3.0.0.

Comment (moljac) [reply config.json]: reverted

File: source/com.squareup.okio/okio/Additions/Options.cs [hunk 2]
```diff
@@ -2,6 +2,9 @@ namespace Square.OkIO
 {
     partial class Options
     {
+        /*
```
Comment (jpobst) [inline source/com.squareup.okio/okio/Additions/Options.cs:5]: Need to revert this change.

File: source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml [hunk 3]
```diff
@@ -76,5 +76,41 @@
...
+        path="/api/package[@name='org.chromium.base.natives']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Base.Natives
+    </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.native_test']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Native_Test
```
Comment (jpobst) [inline source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml:113]: Should this be `NativeTest`?  I don't have an opinion.

**Review by jpobst — COMMENTED** [review]
I think the Okio changes are breaking CI, so I can't review diffs yet.

File: source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml (hunk 3 above)
Comment (moljac) [reply source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml:113]: I was scratching my head. Also lack of opinion. So i said, let's try it this way. From my experience, such namespaces will be removed soon anyway.

File: source/com.squareup.okio/okio/Additions/OkBuffer.cs [hunk 4]
```diff
@@ -4,6 +4,9 @@ namespace Square.OkIO
 {
```
[the remark this answers is not in the document]
Comment (moljac) [reply source/com.squareup.okio/okio/Additions/OkBuffer.cs:4]: yup. forgot to revert that.

**Review by jpobst — APPROVED** [review]
